sanic/websocket.py

```diff
@@ -137,9 +137,11 @@ def __init__(
         self,
         send: Callable[[ASIMessage], Awaitable[None]],
         receive: Callable[[], Awaitable[ASIMessage]],
+        subprotocols: List[str]=[],
```

Comment:
This should default to `None`. If there needs to be a default list, that should be handled inside the function. [See rationale](https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments)

```
subprotocols: Optional[List[str]]=None

...

subprotocols = subprotocols or []
```

Reply:
Updated, thanks for the advice!